core/src/main/java/com/github/mizool/core/converter/IdentifierConverter.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2017-2018 incub8 Software Labs GmbH
+ * Copyright 2017-2018 protel Hotelsoftware GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.mizool.core.converter;
+
+import com.github.mizool.core.Identifiable;
+import com.github.mizool.core.Identifier;
+
+public class IdentifierConverter
+{
+    public String fromPojo(Identifier<?> pojo)
+    {
+        String record = null;
```

Comment:
`com.github.mizool.core.converter` now mixes classes that talk about `result`, `record` and `dto`.

We should think about either breaking it up into sub packages for store and rest layers or try to use the term `result` everywhere, as long as the respective helper class can be used for both layers.

Any thoughts, @bannmann ?

Reply:
Let's rename all references to dto/record in this package to `result`. Some classes here are indeed only used in certain layers, but they should not care about that until we find a real clash in either naming or return/argument types.

Reply:
oh, and all method parameters in this package should be named in a neutral way as well, e.g. `String value` instead of `String record` or `String dto`. Referencing "pojo" is of course okay.